androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/components/CardChip.kt

```diff
@@ -0,0 +1,58 @@
+package com.EENX15_22_17.digital_journal.android.ui.components
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun CardChip(
+    modifier: Modifier = Modifier,
+    value: String,
+    color: Color,
+    shape: Shape = RectangleShape,
+) {
+    Card(
```

Comment:
Any reason for not using the [Composable standard Chip](https://developer.android.com/reference/kotlin/androidx/compose/material/package-summary#Chip(kotlin.Function0,androidx.compose.ui.Modifier,kotlin.Boolean,androidx.compose.foundation.interaction.MutableInteractionSource,androidx.compose.ui.graphics.Shape,androidx.compose.foundation.BorderStroke,androidx.compose.material.ChipColors,kotlin.Function0,kotlin.Function1))?

Reply:
One reason is because the documentation says "This material API is experimental and is likely to change or to be removed in the future." And the other reason is that Chip require a *onClick* method which i didn't like :)

Reply:
Ok! Reasonable enough, even though I think they will stabilize over time rather than be removed (we have already used them in `RettsSymtomChip`)